src/wp-includes/class-wp-scripts.php

```diff
@@ -1073,7 +1139,7 @@ private function has_only_delayed_dependents( $handle, $async_only = false, $che
 	 * @return string $strategy The best eligible loading strategy.
 	 */
 	private function get_eligible_loading_strategy( $handle ) {
-		if ( ! isset( $this->registered[ $handle ] ) ) {
+		if ( ! array_key_exists( $handle, $this->registered ) ) {
```

Comment:
https://github.com/WordPress/wordpress-develop/pull/4391/files?file-filters%5B%5D=.php&show-viewed-files=true#r1232052943

Reply:
Done in 47b2cbccfc7641d141994c5ec0a28e356d12da3e